src/coreclr/src/inc/corinfo.h

```diff
@@ -1321,6 +1321,7 @@ struct CORINFO_LOOKUP_KIND
 //
```

Comment:
This is needs JIT interface GUIID change

Reply:
Note the JIT team requested holding off checking in JitInterface changes: https://github.com/dotnet/runtime/pull/1378#issuecomment-580944654

Pull requests #1378, #2249, #31686 are currently waiting for the window to reopen again.

Cc @BruceForstall on whether the window is going to open soon.

Reply:
Thanks for the heads-up Michal

Reply:
We're ready to take JIT-EE interface changes now (some internal processes dependent on the JIT-EE interface GUID are now stable).

Please wait until #2249 is merged, then update the JIT-EE interface GUID, then feel free to merge.

Reply:
@fadimounir How close is this to getting merged? Have you updated the JIT-EE interface GUID?

Reply:
@BruceForstall I have a couple of code review feedback workitems that I still need to work on. It'll probably be a day or two. Were you trying to merge other PRs and waiting on this one?

Reply:
Looks like https://github.com/dotnet/runtime/pull/31686 got closed without merging, so I guess is abandoned. This makes this the last known JIT-EE interface change in flight.

I was just hoping to batch all the JIT-EE interface changes quickly. But, of course, take the time to get it right.